src/components/tests/organizations.test.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
```

Comment:
I've found it to be a learning curve but I think most of the difficulty is because our components are too big and integrated to begin with.  It will work better with smaller components that are more independent and modular.

It does seem to be harder to test the business logic in the context, but I think that is just a different learning curve that will involve mocking child components.  Most of the logic in the admin panel boils down to "did the api request work", so I think there may be other ways to test that than react-testing-library.  Regular Jest tests are still compatible.

We are also not planning on having these tests replace the cypress integration tests.  Once the components are broken down these tests should work more as modular/unit tests for development.  As it is I can see some overlap between the testing-library tests and the ones we have in cypress.  

I do like that these tests will be independent of code implementation once they're implemented well.  

For the meantime I aim to try breaking down the tests and organizing them by component and improve the cypress integration tests as well.

Reply:
Yes, the contexts are currently replacing the models for our state management.

The views are to help organize the code into component trees that will share the same state context.  They already existed as wrapper components that the router to reference to render each page in the admin panel.  We just wanted to put them together for consistency, and because it'll be easier to add context and any other logic needed by the child components at those root files, etc.